src/org/omegat/util/TMXWriter2.java

```diff
@@ -163,6 +165,24 @@ public void writeEntries(final Map<String, ? extends ITMXEntry> entries) throws
         }
     }
 
+    /**
+     * Write entries with alternative translations.
+     * @param entries Map of EntryKey and TMXEntry to output.
+     */
+    public void writeEntriesAlt(final Map<EntryKey, ? extends ITMXEntry> entries) throws Exception {
```

Comment:
``wirteEntries`` and ``writeEntriesAlt`` can be consolidated. We should refactoring an interface.